github3/pulls.py

```diff
@@ -178,10 +177,8 @@ def _update_attributes(self, pull):
         #: GitHub.com url for review comments (not a template)
         self.review_comments_url = pull.get('review_comments_url')
 
-        m = match('https?://[\w\d\-\.\:]+/(\S+)/(\S+)/(?:issues|pull)?/\d+',
-                  self.issue_url)
         #: Returns ('owner', 'repository') this issue was filed on.
-        self.repository = m.groups()
+        self.repository = self.base.repo
```

Comment:
It's currently [a tuple](https://github.com/sigmavirus24/github3.py/blob/develop/github3/pulls.py#L50), although the `repo` property on `PullDestination` doesn't seem to be documented.